src/SprykerEco/Zed/Econda/Persistence/EcondaQueryContainerInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Persistence;
+
+use Spryker\Zed\Kernel\Persistence\QueryContainer\QueryContainerInterface;
+
+interface EcondaQueryContainerInterface extends QueryContainerInterface
+{
```

Comment:
No `public` methods are available?  Is this on purpose? or just a _placeholder_?

Reply:
@alex-galych what should I change here?

Reply:
@gerner-spryker please advice

Reply:
There is no harm here, the project could already extend it and it will be open for extension. It's fine like this.